utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -460,17 +463,17 @@ private fun calculateInitialMargin(availableAreaWidth: Int, drawableWidth: Float
   return margin.coerceAtLeast(0f)
 }
 
-// Reference: https://stackoverflow.com/a/51865494
-private fun TextView.width(computeWidthOnGlobalLayout: (Int) -> Unit) {
-  if (width == 0) {
+// Reference: https://stackoverflow.com/a/57495622/12314934
+private fun View.afterLayout(what: () -> Unit) {
+  if (isLaidOut) {
```

Comment:
Perhaps refactor the existing one to match the new solution (they seem almost identical, so I don't know why it's being rearranged so much).